validation/src/test/scala/hmda/validation/rules/lar/validity/RaceEditCheckSpec.scala

```diff
@@ -0,0 +1,57 @@
+package hmda.validation.rules.lar.validity
+
+import hmda.model.fi.lar.{ Applicant, LoanApplicationRegister }
+import hmda.validation.rules.lar.LarEditCheckSpec
+
+abstract class RaceEditCheckSpec extends LarEditCheckSpec {
+
+  def succeedsWithRace(vals: List[String]) = {
+    forAll(larGen) { lar: LoanApplicationRegister =>
+      val testApplicant = applicantWithRaceVals(lar.applicant, vals)
+      val testLar = lar.copy(applicant = testApplicant)
+      testLar.mustPass
+    }
+  }
+  def succeedsWithCoRace(vals: List[String]) = {
+    forAll(larGen) { lar: LoanApplicationRegister =>
+      val testApplicant = applicantWithCoRaceVals(lar.applicant, vals)
+      val testLar = lar.copy(applicant = testApplicant)
+      testLar.mustPass
+    }
+  }
+
+  def failsWithRace(vals: List[String]) = {
+    forAll(larGen) { lar: LoanApplicationRegister =>
+      val testApplicant = applicantWithRaceVals(lar.applicant, vals)
+      val testLar = lar.copy(applicant = testApplicant)
+      testLar.mustFail
+    }
+  }
+  def failsWithCoRace(vals: List[String]) = {
```

Comment:
I had pretty much the same assessment of the pros and cons.

- Pros of the list: I knew I would be passing the values to a helper method (`applicantWithRaceVals`), and passing five named variables straight through seemed ugly and a hassle.
- Pros of named variables: enforce correct number of values and correct types (race1 = `Int`, others = `String`).

Another issue with the current implementation is that it only allows hard-coded values. Maybe switching to named variables would make it easier to use a generator to fill in the desired field with randomized values...